app/controllers/games_controller.rb

```diff
@@ -12,4 +12,10 @@ def show
     @game = Game.find(params[:id])
     @chess_pieces = @game.chess_pieces.order(:position_y).order(:position_x).to_a
   end
+
+  def forfeit
```

Comment:
nevermind ignore me. haven't had coffee yet. Didn't notice this was a controller. :)